src/common/tse_internal.h

```diff
@@ -33,8 +33,8 @@
  * Author: Di Wang  <di.wang@intel.com>
  */
 
-/* NB: tse_task_private is TSE_PRIV_SIZE = 1016 bytes for now */
-#define TSE_TASK_ARG_LEN		888
+/* NB: tse_task_private is TSE_PRIV_SIZE = 1152 bytes for now */
+#define TSE_TASK_ARG_LEN		1024
```

Comment:
I suspected the tse/stack size today, but did not find problem here.
Why do you enlarge the size? did you find the stack/embedded buf overflow? The existed logic should be able to detect the overflow now.
But with your this new version patch, I cannot reproduce the 2846 segfault (ran 3 ~ times)

Reply:
Honestly, I just suspected the stack overflow, but I cannot find the point. So I enlarge the stack buffer for try. It seems works. I will split this part into another separated patch PR#817, then if we found the root reason or more suitable solution, we can replace such patch. But currently, if without such fix, this patch and subsequent patch (PR#782) will always hit DAOS-2846 failure.

Reply:
Sorry, enable stack buffer size does not resolve DAOS-2846.